### ISSUE TITLE
Consider grouping Pie Chart slices

### ISSUE THREAD

**Issue by beto-rodriguez** [beto-rodriguez/Live-Charts#447]
It would be nice to group all the small pie slices when there are bellow a previous set value.

![pie](http://i.imgur.com/bcCQuY7.png)

#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>*</td>
</tr>
<tr>
    <td>.Net Version</td>
    <td>*</td>
</tr>
<tr>
    <td>Windows</td>
    <td>*</td>
</tr>
</table>

**Comment by hyankov** [comment]
It would be nice if it's an option. I imagine it as:

`<LiveCharts:PieChart CombineSlicesUnderThreshold="True" CombineThresholdParticipation="3" CombinedSlicesLabel="Other, insignificant" ... />`

Which would then combine all slices which are under 3% together, into a single slice, with a label "Other, insignificant".

Bonus points for somehow expanding/showing the combined slices on mouse hover ;)

**Comment by beto-rodriguez** [comment]
Well I was able to build a sample with the current version of the library, I only added a new extension `.AsSeriesCollection` just to make it more lINQ firendly, but besides that I didn't changed anything, anyways I'll add a new sample to the site showing this feature.

It seems complicated, but I think this is a good start, in a future we can make it easier.

https://github.com/beto-rodriguez/Live-Charts/tree/develop/Examples/Wpf/PieChart/DropDowns

And the result: here I am grouping the small slices in this black slices, then when I click over them I show the content, when I click "Go back" I display the previous state.

![dropdown tooltip](https://cloud.githubusercontent.com/assets/10853349/23493029/5d74fa0a-fece-11e6-8ba9-9c4caea47302.gif)
